### ISSUE TITLE
[DOCS] Website

### ISSUE THREAD

**Issue by isaacs** [vltpkg/vltpkg#45]
Make a website with API and usage docs for `vlt`.

Technical documentation, module APIs, authoritative configuration descriptions, and so on, should be generated from the code as much as possible. `pnpm typedoc` in the root of the project will create a combined typedoc for everything in the monorepo, so use descriptions in tsdoc comments as much as possible. When the site goes public, it'll be built at https://vltpkg.github.io/vltpkg. (Might want to limit this to a specific branch or something?) We can also rely on this to not have to write a comprehensive list of functions in every README. Just a usage block is enough. But once the API docs site is live, we should link all the READMEs to their page in it, so that people can find it from npm, github, etc.

The website should include this (or just link to it prominently), but focus on higher level concepts and usage instructions, linking to the relevant API doc for types, option/function descriptions, etc.

tl;dr -

- keep writing good tsdocs
- [x] design a website
- [x] build a website (link to API docs a lot)
- [ ] write content for the website
- [x] figure out deployment etc (wire up to release tags or something)

**Comment by lukekarrys** [comment]
https://docs.vlt.sh is done and deployed. I am going to create follow-up issues for additional content.
